### PR TITLE
richtext: unselect text when cursor is outside draft editor

### PR DESCRIPTION
**Pull request by helenphan0**

### CODE REVIEW

**Comment by GuruM** [comment]
Hmm, which problem does this solve?

**Comment by helenphan0** [comment]
When the draftjs loses its selection state, you can't tell (text highlight remains on the wysiwyg). So, when you try to delete the highlighted text, write over it, or apply a style to it, the changes don't apply to the highlighted text. This fix will blur the editor when the selection state is lost, so it's more apparent that the selection state is gone, and hopefully result in fewer unexpected behavior while editing text.

**Comment by GuruM** [comment]
Ah okay, works as intended then! I'm not sure how easy it would be to figure out how to get draft to retain the selection state, as that would be most ideal (maybe a phase 2 thing), but this definitely prevents unexpected behaviour. 💯

File: src/components/Zone.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class Zone extends React.Component {
       display: 'inline-block',
       margin: `0 -${ props.basePadding }px`,
       padding: `0 ${ props.basePadding }px`,
-      width: `calc(100% + ${ props.basePadding * 2 }px)`,
+      width: `calc(100% + ${ props.basePadding * 2 }px - 2px)`,
```
Comment (GuruM) [inline src/components/Zone.js]: so I think this causes the yellow highlight to appear shifted to the left:
image

(notice how the yellow highlight is on the outside of the editor on the left side, but on the inside on the right side

**Comment by GuruM** [comment]
🎎

**Pull request merged**